bugbug/repository.py

```diff
@@ -20,6 +23,16 @@
 BUG_PATTERN = re.compile('[\t ]*[Bb][Uu][Gg][\t ]*([0-9]+)')
 
 
+class hgclient_template(hgclient):
```

Comment:
Instead of subclassing, we can just create a hg_log function, it's more maintainable as we don't need to worry about the parent at all (e.g. even if upstream changes the constructor, we won't have to update anything if we don't subclass).